docs/en/w4a16.md

```diff
@@ -62,10 +62,14 @@ Memory (GB) comparison results between 4-bit and 16-bit model with context size
 | Llama-2-7B-chat  | 15.1        | 6.3        | 16.2        | 7.5        |
 | Llama-2-13B-chat | OOM         | 10.3       | OOM         | 12.0       |
 
+```
+pip install nvidia-ml-py
+```
+
 ```shell
-python benchmark/profile_generation.py \
-  ./workspace \
-  --concurrency 1 --input_seqlen 1 --output_seqlen 512
+python profile_generation.py \
```

Comment:
Is it wrong with `python benchmark/profile_generation.py`?
If changing it to `python profile_generation.py`, we'd better add `cd benchmark`

Reply:
> Is it wrong with `python benchmark/profile_generation.py`? If changing it to `python profile_generation.py`, we'd better add `cd benchmark`

It works with `benchmark/profile_generation`, I will update it